docs/tutorials/odc-stac.rst

```diff
@@ -0,0 +1,300 @@
+
+.. code-block:: python
+
+   start_date = "2017-01-01"
+   end_date = "2023-01-01"
+   date_range = (start_date, end_date)
+
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+
+Catalogs, collections, and items
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Many Earth observation data providers generate STAC metadata, which can be used to find and load data you're interested in.
+STAC metadata has four important components:
+
+* **Catalog**: A structure for organising multiple datasets managed by a given provider. For example, `Planetary Computer's Catalog <https://radiantearth.github.io/stac-browser/#/external/planetarycomputer.microsoft.com/api/stac/v1/>`_
+* **Collection**: A structure for organising all items in a single dataset. For example, `Land Use Land Cover Collection <https://radiantearth.github.io/stac-browser/#/external/planetarycomputer.microsoft.com/api/stac/v1/collections/io-lulc-annual-v02>`_
+* **Item** A single spatio-temporal item, such as one observation in a dataset. For example, `Land Use Land Cover Data for Supercell 15M in 2013 <https://radiantearth.github.io/stac-browser/#/external/planetarycomputer.microsoft.com/api/stac/v1/collections/io-lulc-annual-v02/items/15M-2023>`_
+* **Asset** A single data measurement associated with an item, such as a single band. The Land Use Land Cover Dataset has only one asset, called "data".
+
+We must specify the URL for the catalog we want to search, along with the desired collection (:code:`io-lulc-annual-v02`) and asset (:code:`data`). 
+The precise items that we need to load will be returned by a query that we run later.
+
+Type the following into the empty cell below the **Catalogs, collections, and items** heading:
+
+.. code-block:: python
+
+   catalog_url = "https://planetarycomputer.microsoft.com/api/stac/v1/"
+   desired_collections = ["io-lulc-annual-v02"]
+   desired_assets = ["data"]
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+
+Connect to catalog and find items
+---------------------------------
+
+We use `pystac-client`_'s :code:`Client` class to connect to Planetary Computer's STAC catalog.
+We also use :code:`planetary_computer.sign_inplace` to authorise our connection.
+Type the following into the empty cell below the **Connect to catalog and find items** heading:
+
+.. code-block:: python
+
+   stac_client = Client.open(
+      url=catalog_url, 
+      modifier=planetary_computer.sign_inplace,
+   )
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+
+Search for items
+^^^^^^^^^^^^^^^^
+
+After setting up the :code:`Client`, we use the :code:`search` method to find items that match our chosen collection, area of interest, and date range.
+Type the following into the empty cell below the **Search for items** heading:
+
+.. code-block:: python
+
+   items = stac_client.search(
+       collections=desired_collections,
+       intersects=aoi_geometry,
```

Comment:
Should the `desired_` prefix be added to the names of the other search parameters as well for consistency?